src/Deserializers/LegacySiteLinkListDeserializer.php

```diff
@@ -65,6 +65,11 @@ private function assertElementIsValidArray( array $arrayElement ) {
 		}
 	}
 
+	/**
+	 * @param array $siteLinkArray
+	 *
+	 * @return SiteLinkList
+	 */
 	private function getDeserialized( array $siteLinkArray ) {
```

Comment:
That would have justified the comment more yes. Not entirely sure about trying to document this kind of stuff though, due to limitations of the doc syntax it won't capture the full thing anyway.